batchflow/models/tf/base.py

```diff
@@ -1940,9 +1941,13 @@ def se_block(cls, inputs, ratio, name='se', **kwargs):
         with tf.variable_scope(name):
             data_format = kwargs.get('data_format')
             in_filters = cls.num_channels(inputs, data_format)
+            if isinstance(kwargs.get('activation'), list) and len(kwargs.get('activation')) == 2:
+                activation = kwargs.pop('activation')
+            else:
+                activation = [tf.nn.relu, tf.nn.sigmoid]
```

Comment:
This ignores the passed activation while it should not

Reply:
it ignores the passed activation only in case it wasn't supposed to be used in the se_block. Two lines above I check whether activation is a list of two elements since se_block needs two.
By default, se_block will get activation from common config, therefore, activations were hardcoded before. Mobilenetv3 needs to change tf.nn.sigmoid to piece-wise linear analog.

Reply:
The passed activation will be ignored only in case it wasn't supposed to be used in se_block. Two lines above I check whether activation is a list of two elements needed for se_block. 
Se_block can get an activation function shared with other layers, for example, in the current resnet use case.

Reply:
If the passed activation is not a list of two, it will be silently ignored at all. Which is not a good idea. You could have a default list of `[tf.nn.relu, tf.nn.sigmoid]` and replace the first item with the passed activation. Or you could throw an exception. Or a warning.

Reply:
added warning

Reply:
Why not `activation = [activation, tf.nn.sigmoid]`?

Reply:
Because it is not enough for the current case and may show bad behavior if it is passed here as default for other blocks.

Reply:
done.